lib/ecto/repo/schema.ex

```diff
@@ -899,9 +899,13 @@ defmodule Ecto.Repo.Schema do
       {:ok, value} ->
         value
       :error ->
+        type_str = case type do
+          {:parameterized, t, _} -> t
+          _ -> type
+        end
```

Comment:
I would revert this change simply because in all other errors it will show up as `{:paramterized, ..., ...}`, so I wouldn't sweat about it for now.